ModPatches/Save Our Ship 2/Patches/Save Our Ship 2/Hediffs_SOS2.xml

```diff
@@ -17,7 +17,7 @@
 	<Operation Class="PatchOperationReplace">
 		<xpath>Defs/HediffDef[defName="SoSArchotechSkin"]/stages/li/statOffsets/ArmorRating_Sharp</xpath>
 		<value>
-			<ArmorRating_Sharp>8</ArmorRating_Sharp>
+			<ArmorRating_Sharp>16</ArmorRating_Sharp>
```

Comment:
Being in mind that this is going to stack with other sources of armor, I'd drop this to 12 - 14mm, similar to FSF above.